example-playbooks/replace_node/replace_node.yml

```diff
@@ -192,13 +192,14 @@
         port: "{{ cql_port }}"
         host: "{{ listen_address }}"
         timeout: "{{ new_node_bootstrap_wait_time_sec }}"
+      register: _wait_for_bootstrap
```

Comment:
I don't see where it's used.

Reply:
It's not. This was a leftover from another test I was doing.
It's removed now.